### PR TITLE
[onert] Add one_op_tests of REVERSE for acl_cl backend

### PR DESCRIPTION
**Pull request by hasw7569**
related pr #4152 
add one_op_tests of REVERSE_V2 for acl_cl,cpu backend

### CODE REVIEW

**Comment by hseok-oh** [comment]
```
13:46:37  [  FAILED  ] 2 tests, listed below:
13:46:37  [  FAILED  ] GenModelTest.OneOp_REVERSEV2_3D
13:46:37  [  FAILED  ] GenModelTest.OneOp_REVERSEV2_1D
```

**Comment by hseok-oh** [comment]
It fails on debug build only. Maybe validation check in armcompute fail. Please check armcompute validation check for resverse.

**Comment by hseok-oh** [comment]
```
14:00:27  Error during model prepare : ResizeBilinear: size value must be positive value, output_height = -3
14:00:27  [       OK ] GenModelTest.neg_OneOp_ResizeBilinear_InvalidSizeVal (2 ms)
14:00:27  [ RUN      ] GenModelTest.OneOp_REVERSEV2_3D
14:00:27  Error during model prepare : in validate_arguments src/core/CL/kernels/CLReverseKernel.cpp:45: ITensor data type S32 not supported by this kernel
14:00:27  /opt/jenkins_slave/workspace/nnfw/master/pr-cross-runtime-test-pipeline/tests/nnfw_api/src/GenModelTest.h:286: Failure
14:00:27        Expected: (nnfw_prepare(_so.session))
14:00:27        Which is: 1
14:00:27  To be equal to: NNFW_STATUS_NO_ERROR
14:00:27        Which is: 0
14:00:27  [  FAILED  ] GenModelTest.OneOp_REVERSEV2_3D (6 ms)
14:00:27  [ RUN      ] GenModelTest.OneOp_REVERSEV2_1D
14:00:27  Error during model prepare : in validate_arguments src/core/CL/kernels/CLReverseKernel.cpp:45: ITensor data type S32 not supported by this kernel
14:00:27  /opt/jenkins_slave/workspace/nnfw/master/pr-cross-runtime-test-pipeline/tests/nnfw_api/src/GenModelTest.h:286: Failure
14:00:27        Expected: (nnfw_prepare(_so.session))
14:00:27        Which is: 1
14:00:27  To be equal to: NNFW_STATUS_NO_ERROR
14:00:27        Which is: 0
```

**Comment by YiHyunJin** [comment]
@nnfw-bot test cross-runtime-test-debug

**Comment by hasw7569** [comment]
@nnfw-bot test cross-runtime-test-release

**Review by YiHyunJin — APPROVED** [review]
LGTM

**Review by intom — APPROVED** [review]
LGTM

File: tests/nnfw_api/src/one_op_tests/Reverse.cc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "GenModelTest.h"
+
+TEST_F(GenModelTest, OneOp_REVERSEV2_3D)
```
Comment (hseok-oh) [inline tests/nnfw_api/src/one_op_tests/Reverse.cc:19]: Please rename test: `OneOp_ReverseV2_3D`, OneOp_ReverseV2_1D`

Comment (hasw7569) [reply tests/nnfw_api/src/one_op_tests/Reverse.cc]: modified

**Review by hseok-oh — APPROVED** [review]
LGTM

**Review by YiHyunJin — APPROVED** [review]

**Comment by hasw7569** [comment]
@intom plz approve it.

**Review by intom — APPROVED** [review]
LGTM

**Pull request merged**